zschema/keys.py

```diff
@@ -30,6 +30,27 @@ def __cmp__(self, other):
             return cmp(int(self.port), int(other.port))
 
 
+# Create a dict with all the keys / values of rhs and lhs, where values
+# from lhs are used if a key is in both.
+def left_merge(lhs, rhs):
+    return dict(v for v in rhs.items() + lhs.items())
```

Comment:
This is an unrecommended solution for merging dictionaries.nThere's a long write up about why here: https://stackoverflow.com/questions/38987/how-to-merge-two-dictionaries-in-a-single-expression

Reply:
I think that's leftover from when I had it inlined as a single expression. Since it's a separate function, copy/update is superior in literally every way. Fixing.